operator/v1/types_ingress.go

```diff
@@ -187,6 +188,14 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	HTTPHeaderBuffer IngressControllerHTTPHeaderBuffer `json:"httpHeaderBuffer,omitempty"`
+
+	// unsupportedConfigOverrides allows specifying unsupported
+	// configuration options.  Its use is unsupported.
+	//
+	// +optional
+	// +nullable
+	// +kubebuilder:pruning:PreserveUnknownFields
+	UnsupportedConfigOverrides runtime.RawExtension `json:"unsupportedConfigOverrides"`
```

Comment:
Forgive my ignorance, but does the `// +nullable` mean we don't need an `omitempty` json flag?

Reply:
I copied this from https://github.com/openshift/api/blob/master/operator/v1/types.go#L78.  I don't know why `omitempty` is omitted.